k8s/controller/autoscale_job/trainer.yaml

```diff
@@ -17,7 +17,7 @@ spec:
       containers:
       - name: trainer
         image: paddlepaddle/paddlecloud-job
-        imagePullPolicy: Always
+        imagePullPolicy: Never
```

Comment:
Curious why it's "Never"?

Reply:
Oh, That's for my local running, will update.